src/utils/error-handling.js

```
@@ -327,8 +327,12 @@ export async function resilientFetch(url, options = {}, retryOptions = {}) {
 
       // Check if response is OK
       if (!response.ok) {
+        let errorText = "";
+        try {
+          errorText = await response.text();
+        } catch (e) {}
```

ESLint

[error] Empty block statement.

(no-empty)